scripts/pyproject.toml

```diff
@@ -0,0 +1,33 @@
+[tool.towncrier]
+    package = "sanic"
```

Comment:
I don't think .toml need indentations..

Reply:
@yunstanford Yes, it does not. However, I personally believe that indenting thigs make it more readable and clean to look at. I can always drop the indentation if you think that should go away